features/add_dish_to_basket.feature

```diff
@@ -1,19 +1,21 @@
 Feature: As a Visitor
   In order to place an order.
-  I need to be able to add dishes to a basket.
+  I need to be able to add dishes to an order.
 
   Background:
-    Given I have a dish called "Meatballs" with price "8.99"
-    And I have a dish called "Salad" with price "4.99"
-
-  Scenario: Visitor adds first dish to basket
+    Given I have a dish called "Garlic Bread" with price "3.99" and category "Starter"
+    And I have a dish called "Salad" with price "4.99" and category "Main Course"
+    And I have a dish called "Meatballs" with price "8.99" and category "Main Course"
+    And I have a dish called "Brule" with price "5.99" and category "Dessert"
```

Comment:
`I have a dish ` ?? really? Do we mean if there is a dish like that in the database or if the user has one? Who is `I`?
Also, I would suggest squashing those 4 steps into one using a table.